### PR TITLE
Update coco_eval.py

### PR DESCRIPTION
**Pull request by LLeeSG**
all the subsequent bbox of current image  will be ignored incorrectly if the conditional in the line 314 met

### CODE REVIEW

File: ppdet/utils/coco_eval.py [hunk 1]
```diff
@@ -311,7 +311,9 @@ def bbox2out(results, clsid2catid, is_bbox_normalized=False):
             for j in range(num):
                 dt = bboxes[k]
                 clsid, score, xmin, ymin, xmax, ymax = dt.tolist()
-                if clsid < 0: continue
+                if clsid < 0:
```
Comment (qingqing01) [inline ppdet/utils/coco_eval.py:314]: 图片

Please use pre-commit scripts to auto-format your code

**Review by qingqing01 — COMMENTED** [review]
Thanks!

File: ppdet/utils/coco_eval.py (hunk 1 above)
Comment (qingqing01) [reply ppdet/utils/coco_eval.py:314]: 1. 自己终端 pip install pre-commit
2. PaddleDetection的根目录下： pre-commit install
3. 把改动的文件，稍微改下。 重新git commit,  自动格式化。
4. 再push下

Comment (LLeeSG) [reply ppdet/utils/coco_eval.py]: 3q, got it done！is it ok now?

**Review by jerrywgz — APPROVED** [review]
Thanks, LGTM

**Pull request merged**